docs/_data/options.js

```diff
@@ -1,178 +1,201 @@
 	{
-		name: ' --multiphp',
-		id: 'multiphp',
-		param: '--multiphp',
-		desc: 'Allows installing multiple PHP versions.',
-		selected: true,
+		flag: 'proftpd',
+		label: 'ProFTPD',
+		description: 'Advanced, modular FTP server that supports LDAP.',
+		default: 'no',
 	},
 	{
-		name: ' --vsftpd',
-		id: 'vsftpd',
-		param: '--vsftpd',
-		desc: 'Lightweight, minimalist and secure FTP server.',
-		selected: true,
-		conflicts: 'proftpd',
+		flag: 'named',
+		label: 'BIND',
+		description: 'Custom DNS name server.',
+		default: 'yes',
 	},
 	{
-		name: ' --proftpd',
-		id: 'proftpd',
-		param: '--proftpd',
-		desc: 'Advanced, modular FTP server that supports LDAP.',
-		selected: false,
-		conflicts: 'vsftpd',
+		flag: 'mysql',
+		label: 'MariaDB',
+		description: 'Fork of MySQL with additional features and improvements.',
+		default: 'yes',
 	},
 	{
-		name: ' --named',
-		id: 'named',
-		param: '--named',
-		desc: 'Custom DNS name server.',
-		selected: true,
+		flag: 'mysql8',
+		label: 'MySQL 8',
+		description: 'Open-source database system.',
+		default: 'no',
 	},
 	{
-		name: ' --mariadb',
-		id: 'mariadb',
-		param: '--mariadb',
-		desc: 'Fork of MySQL with additional features and improvements.',
-		selected: true,
-		conflicts: 'mysql8',
+		flag: 'postgresql',
+		label: 'PostgreSQL',
+		description: 'Open-source database system.',
+		default: 'no',
 	},
 	{
-		name: ' --mysql8',
-		id: 'mysql8',
-		param: '--mysql8',
-		desc: 'Open-source database system.',
-		selected: false,
-		conflicts: 'mariadb',
+		flag: 'exim',
+		label: 'Exim',
+		description: 'Allows sending emails from web-mail or via SMTP.',
+		default: 'yes',
 	},
 	{
-		name: ' --postgresql',
-		id: 'postgresql',
-		param: '--postgresql',
-		desc: 'Open-source database system.',
-		selected: false,
+		flag: 'dovecot',
+		label: 'Dovecot',
+		description: 'Receive emails and connect with email clients via IMAP/POP3.',
+		default: 'yes',
 	},
 	{
-		name: ' --exim',
-		id: 'exim',
-		param: '--exim',
-		desc: 'Allows sending emails from webmail or via SMTP.',
-		selected: true,
+		flag: 'sieve',
+		label: 'Sieve',
+		description: 'Manage your own custom email filters.',
+		default: 'no',
 	},
 	{
-		name: ' --dovecot',
-		id: 'dovecot',
-		param: '--dovecot',
-		desc: 'Receive emails and connect with email clients via IMAP/POP3.',
-		selected: true,
-		depends: 'exim',
+		flag: 'clamav',
+		label: 'ClamAV',
+		description: 'Scans your email inbox for viruses.',
+		default: 'yes',
 	},
 	{
-		name: ' --sieve',
-		id: 'sieve',
-		param: '--sieve',
-		desc: 'Manage your own custom email filters.',
-		selected: false,
-		depends: 'dovecot',
+		flag: 'spamassassin',
+		label: 'SpamAssassin',
+		description: 'Filter out spam emails from your inbox.',
+		default: 'yes',
 	},
 	{
-		name: ' --clamav',
-		id: 'clamav',
-		param: '--clamav',
-		desc: 'Scans your email inbox for viruses.',
-		selected: true,
-		depends: 'exim',
+		flag: 'iptables',
+		label: 'iptables',
```

Comment:
I assume you used the default "spelling"  it looks odd but

Reply:
Yeah `iptables` seems to be the correct casing 🤷